api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -28,5 +28,7 @@ private FhirConstants() {
 	public static String OPENMRS_FHIR_EXT_NAME = OPENMRS_FHIR_EXT_PREFIX + "/name";
 	
 	public static String PATIENT = "Patient";
+
+	public static String PERSON_ATTRIBUTE_TYPE_PROPERTY = "fhir2.personAttributeTypeUuid";
```

Comment:
I'm thinking about this from a maintenance perspective and I think it might be better if we constructed the name of the global property the same everywhere, so here and in `config.xml`. The way to do this would be to add a line to `fhir.properties` like so:

```properties
project.parent.artifactId=@project.parent.artifactId@
```

Then annotating this like this:

```java
@Value("${project.parent.artifactId}.personAttributeTypeUuid")
public static String PERSON_ATTRIBUTE_TYPE_PROPERTY;
```

Does this seem like a good idea to you?

Reply:
Yeah. It is ☺️